src/Hosting.CommandLine/HostExtensions.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Nate McMaster.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.Extensions.Hosting
+{
+    using System;
+    using System.Runtime.ExceptionServices;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using McMaster.Extensions.CommandLineUtils;
+    using McMaster.Extensions.Hosting.CommandLine.Internal;
+    using Microsoft.Extensions.DependencyInjection;
```

Comment:
Style nit-pick: can you move usings to the top of the file instead of nesting inside the namespace? Sorry VS doesn't provide a way to enforce this style consistently :-/

Reply:
Will do.  I think you can set that in .editorconfig

Reply:
Good suggestion! I've added to editorconfig.